arch/arm/soc/nordic_nrf/Kconfig

```diff
@@ -15,5 +15,7 @@ config SOC_FAMILY
 	string
 	default "nordic_nrf"
 
+gsource "arch/arm/soc/nordic_nrf/Kconfig.peripherals"
```

Comment:
A plain `source` would do here. `gsource` works like an optional include (like `-include` in `make`) when you don't have any wildcards in the filename.

Reply:
fixed